eZ/Bundle/EzPublishElasticsearchSearchEngineBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * This file is part of the eZ Publish Kernel package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace eZ\Bundle\EzPublishElasticsearchSearchEngineBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    protected $rootNodeName;
+
+    public function __construct( $rootNodeName )
+    {
+        $this->rootNodeName = $rootNodeName;
+    }
+
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root( $this->rootNodeName );
+
+        $this->addConnectionsSection( $rootNode );
+
+        return $treeBuilder;
+    }
+
+    /**
+     * Adds semantic configuration definition.
+     *
+     * @param \Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition $node
+     */
+    protected function addConnectionsSection( ArrayNodeDefinition $node )
+    {
+        $node->children()
+            ->scalarNode( "default_connection" )
+                ->info( "Name of the default connection" )
+            ->end()
+            ->arrayNode( "connections" )
+                ->info( "Elasticsearch Search Engine connections configuration" )
+                ->useAttributeAsKey( "key" )
```

Comment:
Geat, added in https://github.com/ezsystems/ezpublish-kernel/commit/9d3f0f09c3e43e9378b8a5e81ad60f9207bcc642.